src/cpp/session/CMakeLists.txt

```
@@ -695,15 +697,21 @@ if(NOT RSTUDIO_SESSION_WIN32 AND NOT RSESSION_ALTERNATE_BUILD)
          if (RSTUDIO_ELECTRON)
             install(DIRECTORY "${RSTUDIO_DEPENDENCIES_QUARTO_DIR}" 
                   DESTINATION "${RSTUDIO_INSTALL_RESOURCES}/app"
-                  USE_SOURCE_PERMISSIONS)
+                  USE_SOURCE_PERMISSIONS
+                  PATTERN ".gitignore"
+                  EXCLUDE)
          else()
             install(DIRECTORY "${RSTUDIO_DEPENDENCIES_QUARTO_DIR}"
                   DESTINATION "${RSTUDIO_INSTALL_BIN}"
                   USE_SOURCE_PERMISSIONS
-                  PATTERN "*/share" EXCLUDE)
+                  PATTERN "*/share"
+                  PATTERN ".gitignore"
+                  EXCLUDE)
```

I'm not super familiar with CMake exclude syntax, but does each `PATTERN` need to be followed by `EXCLUDE`? Or maybe the single `EXCLUDE` covers both `PATTERN`s?

```suggestion
                  PATTERN "*/share" EXCLUDE
                  PATTERN ".gitignore" EXCLUDE)
```

---

Based on the example at https://cmake.org/cmake/help/latest/command/install.html#installing-directories, I think you're right -- each exclude pattern needs to be followed by `EXCLUDE`.

---

I think I read those docs wrong. I'll update it!